main.py

```diff
@@ -229,12 +230,15 @@ def create_circuit(self, data):
         except ValueError as exception:
             raise BadRequest(str(exception)) from exception
 
+        # save circuit
+        try:
+            evc.sync()
+        except ValidationError as exception:
```

Comment:
@Alopalao could you also apply this same exc handling for the `PATCH /v2/evc/<circuit_id>` route? 

It turns out that when it's calling `evc.update` that will also call `evc.sync()` it's not handling `ValidationError` but only `ValueError` yet. 

We're still vulnerable to other validation issues there, sicne the `@validate` OpenAPI dec isn't applied there yet (this one here would be covered in a future oportunity)

Reply:
Got it.